rmgpy/rmg/main.py

```diff
@@ -1268,7 +1268,16 @@ def make_seed_mech(self, first_time=False):
             if os.path.exists(seed_dir):  # This is a seed from a previous RMG run. Delete it
```

Comment:
I wonder why you delete the seed from a previous RMG run while you move the seed from a previous iteration to a temporary directory in case you run into errors. Probably I am not fully aware how restart and seed works, but what happens if I want to use a seed from a previous RMG run and do not copy it out of the seed folder? Will it be deleted at this point and my seed from the previous RMG run is gone for good? In this case wouldn't it be good to also just move the seed to a temporary directory?

Reply:
Good question! Currently, if the user restarts from the seed in this folder, we make a copy in a new folder called `previous_restart` so that the seed doesn't get lost. See below.

https://github.com/ReactionMechanismGenerator/RMG-Py/blob/6483418a348ea471b017c0d17b3ec68e00e36127/rmgpy/rmg/main.py#L488-L499


If the user instead were to run a new RMG job from this folder the seed from the previous job would be lost. This was the behavior before I implemented the restart stuff, but I am willing to consider other options if you think this is worth while.